Sources/Datadog/CrashReporting/Integrations/CrashReportingWithRUMIntegration.swift

```diff
@@ -72,8 +72,10 @@ internal struct CrashReportingWithRUMIntegration: CrashReportingIntegration {
             realDateNow: dateProvider.now.addingTimeInterval(currentTimeCorrection)
         )
 
-        if let lastRUMViewEvent = context.lastRUMViewEvent {
-            sendCrashReportLinkedToLastViewInPreviousSession(report, lastRUMViewEventInPreviousSession: lastRUMViewEvent, using: adjustedCrashTimings)
+        if let lastRUMViewEvent = crashContext.lastRUMViewEvent {
+            if (lastRUMViewEvent.view.crash?.count ?? 0 < 1) { // if a cross-platform crash was reported, do not send its native version
+                sendCrashReportLinkedToLastViewInPreviousSession(crashReport, lastRUMViewEventInPreviousSession: lastRUMViewEvent, using: adjustedCrashTimings)
+            }
```

Comment:
to avoid nested if, you can use`,` comma and add a statement directly in the first if let.

```suggestion
        // if a cross-platform crash was reported, do not send its native version
        if let lastRUMViewEvent = crashContext.lastRUMViewEvent, lastRUMViewEvent.view.crash?.count ?? 0 < 1
                sendCrashReportLinkedToLastViewInPreviousSession(crashReport, lastRUMViewEventInPreviousSession: lastRUMViewEvent, using: adjustedCrashTimings)
            }
```